File: Assignment39/Question1.c
#include<stdio.h>
int  Display(char *str)
{
    static int Count=0;
    int i=0;
    if(*str!='\0')
    {
        if(*str==' ')
        {
            Count++;
        }
        str++;
        Display(str);
    }
    return Count;
}
int main()
{
    char arr[20];
    int iret=0;
    printf("Enter String:");
    scanf("%[^\n]s",arr);
    iret = Display(arr);

    printf("sum of white space in string: %d\n",iret);
    return 0;
}

File: Assignment8/Question1.java
import java.util.*;
class Diffrenece
{
    int EvenOdd(int Brr[])
    {
        int iSumEven=0,iSumOdd=0,iDiff=0,iCnt=0;
        for(iCnt=0;iCnt<Brr.length;iCnt++)
        {
            if(Brr[iCnt]%2==0)
            {
                iSumEven=iSumEven+Brr[iCnt];
            }
            else{
                iSumOdd=iSumOdd+Brr[iCnt];
            }
        }
        iDiff = iSumEven-iSumOdd;
        
        return iDiff;
    }
}
public class Question1
 {
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter number of ELement:");
        int iSize = sobj.nextInt();
        
        int Arr[]=new int[iSize];

        System.out.println("Enter Element:");
        for(int iCnt=0;iCnt<Arr.length;iCnt++)
        {
            Arr[iCnt]= sobj.nextInt();  
        }
        Diffrenece dobj = new Diffrenece();
        
        int iRet = dobj.EvenOdd(Arr);

        System.out.println(iRet+" is Diffrence between Sum of Even ans odd");
    }
    
}
